README.md

```diff
@@ -96,4 +99,37 @@ To generate a new encrypted value or to replace one of the provided values, for
 $ ansible-vault encrypt_string --vault-password-file your_password_file --name 'trello_api_key'
 ```
 
-then enter the value you want to encrypt when prompted.
\ No newline at end of file
+then enter the value you want to encrypt when prompted.
+
+## Docker
+
+Run the project:
+```bash
+docker build --tag todo-app .
+docker run -dit \
+    --name todo-app \
+    -p 8000:8000 \
+    --env-file .env \
+    todo-app
+```
+
+Run the project with mounting:
+```bash
+docker build --target development --tag todo-app:dev .
+docker run -dit \
+    --name todo-app:dev \
```

Comment:
Definitely personal preference not recommendation, but I tend to avoid naming my containers because that forces me to be much more disciplined about removing them before running a new one with the same name!